src/Raven.Server/NotificationCenter/Indexing.cs

```diff
@@ -121,7 +141,22 @@ internal void UpdateIndexing(object state)
         {
             try
             {
-                if (_warningIndexOutputsPerDocumentQueue.IsEmpty && _warningReferenceDocumentLoadsQueue.IsEmpty && _mismatchedReferencesLoadWarning == null && _warningComplexFieldAutoIndexing.IsEmpty)
+                if (_cpuExhaustionWarningIndexNames.Count == 0)
+                {
+                    if (_isCpuExhaustionWarningAdded)
+                    {
+                        _notificationCenter.Dismiss(AlertRaised.GetKey(AlertType.Throttling_CpuCreditsBalance, Source));
+                        _isCpuExhaustionWarningAdded = false;
+                    }
+                }
+                else
+                {
+                    var cpuCreditsExhaustionAlertMessage = new CpuCreditsExhaustionWarning(_cpuExhaustionWarningIndexNames);
+                    _notificationCenter.Add(GetCpuCreditsExhaustionAlert(cpuCreditsExhaustionAlertMessage));
+                    _isCpuExhaustionWarningAdded = true;
+                }
+                
+                if (_warningIndexOutputsPerDocumentQueue.IsEmpty && _warningReferenceDocumentLoadsQueue.IsEmpty && _mismatchedReferencesLoadWarning == null && _warningComplexFieldAutoIndexing.IsEmpty && _cpuExhaustionWarningIndexNames.Count == 0)
```

Comment:
I think we shouldn't check `_cpuExhaustionWarningIndexNames` here

Reply:
Fixed